### DISCUSSION TITLE
How to instantly show the tooltip without a fade-in animation

### DISCUSSION THREAD

**Discussion by stefanerwinmayer** [floating-ui/floating-ui#1688] [Q&A] [answered]
Hi all

I'm using Popper v2 with React and I'm creating a tooltip when hovering over an element. How can make it so the tool tip appears instantly without delay that is caused by what seems to be a default fade-in animation?

Thank you for your help.

**Comment by FezVrasta** [comment, marked as answer]
Popper doesn't provide any show/hide logic. Are you probably referring to some 3rd party wrapper such as Material UI?

**Reply by stefanerwinmayer** [reply]
I am indeed handling show/hide myself. What I have observed when showing the tooltip is a delay of a few milliseconds for the tooltip to appear. It looks like some sort of fade-in animation. Can you confirm that Popper is not the source of said delay?

**Reply by FezVrasta** [reply]
Popper only positions the element, it doesn't apply any transition indeed.

**Reply by stefanerwinmayer** [reply]
Thank you @FezVrasta
